src/walk.rs

```diff
@@ -502,6 +503,19 @@ impl WorkerState {
                     }
                 }
 
+                let reference = GeoLocation {
+                    latitude: 47.3,
+                    longitude: 11.3,
+                };
+                let max_distance = 500.;
```

Comment:
The reference location and `max_distance` need to be parsed from the command line ...